src/graph.c

```diff
@@ -224,7 +224,7 @@ static void check_reachability(meshlink_handle_t *mesh) {
 
 			n->status.udp_confirmed = false;
 			n->maxmtu = MTU;
-			n->minmtu = 0;
+			n->minmtu = 500;
```

Comment:
Setting `n->minmtu` to a non-zero value indicates to the rest of the code that UDP communication of packets up to that size is possible, so this is bad. If we want to ensure we don't send MTU probes smaller than 500 bytes, we should just change `send_mtu_probe_handler()` to only send probes bigger than 500 bytes.